modules/oc-compliance-installing.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * security/oc_compliance_plug_in/oc-compliance-plug-in-using.adoc
+
+[id="installing-oc-compliance_{context}"]
+= Installing the oc-compliance plug-in
+
+.Procedure
+
+. Extract the `oc-compliance` image to get the `oc-compliance` binary:
++
+[source,terminal]
+----
+$ mkdir oc-compliance
+----
```

Comment:
@ahardin-rh No need to create directory as well. We can remove this step. Thanks